src/Term.ts

```diff
@@ -13,6 +13,7 @@ class Term {
     directory?: string
   ): Promise<{ status: number; output: string }> {
     const manager = hasYarn() ? "yarn" : "npm";
+    const runner = hasYarn() ? "yarn dlx --quiet" : "npx";
```

Comment:
`yarn dlx` is a Yarn 2 only command so this will fail under Yarn 1, if you're after executing the local version of a dependency you should use `yarn`

Reply:
If there any way to check if Yarn 2 is being used ?

Reply:
Check if `yarn.lock` contains `__metadata:` or run `yarn --version` in the repo